site/en/codelabs/esp-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,600 @@
+
+## Join the thread network via secure commissioning
+
+Duration: 5:00
+
+In this session, we will add Board2 to the network via security commissioning:
+
+Get the PSKc and panid from BR Commissioner:
+
+```console
+
+## BR Commissioner ##
+----------------------
+> pskc
+
+54e7f18d2575014da94db09df29c5df0
+Done
+> panid
+
+0x1444
+Done
+```
+
+Config these network informations to Board2:
+
+
+```console
+
+## Board2 Joiner ##
+----------------------
+
+> dataset pskc 54e7f18d2575014da94db09df29c5df0
+
+Done
+> dataset panid 0x1444
+
+Done
+```
+
+Commit this dataset as the active one:
+
+```console
+
+## Board2 Joiner ##
+----------------------
+> dataset commit active 
+
+Done
+```
+
+Bring up the IPv6 interface:
+
+```console
+
+## Board2 Joiner ##
+----------------------
+> ifconfig up
+
+Done
+I (29146) OT_STATE: netif up
+```
+
+Get the eui64 from Board2:
+
+```console
+## Board2 Joiner ##
+----------------------
+> eui64
+
+4831b7fffec02be1
+Done
+```
+
+On the **BR Commissioner**, start the commissioner and specify the `eui64` of the device that can join, along with the Joiner Credential, for example `J01NME`. The Joiner Credential is a device-specific string of all uppercase alphanumeric characters (0-9 and A-Y, excluding I, O, Q and Z for readability), with a length between 6 and 32 characters.
+
+```console
+
+## BR Commissioner ##
+----------------------
+> commissioner start
+
+Commissioner: petitioning
+Done
+Commissioner: active
+
+> commissioner joiner add 4831b7fffec02be1 J01NME
+
+Done
+```
+
+Switch to the **Board2 Joiner**. Start the joiner role with the Joiner Credential that you just set up on the BR Commissioner:
+
+```console
+## Board2 Joiner ##
+----------------
+
+> ifconfig up
+Done
+> joiner start J01NME
+Done
+```
+
+Within a minute or so, you get a confirmation of a successful authentication:
+
+```console
+## Board2 Joiner ##
+----------------
+
+>
+Join success
+```
+
+After that, you can start and join the thread network formed by BR Commissioner.
+
+Start Thread protocol operation:
+
+```console
+> thread start
+
+I(35727) OPENTHREAD:[N] Mle-----------: Role disabled -> detached
+Done
+> I(36197) OPENTHREAD:[N] Mle-----------: Attach attempt 1, AnyPartition reattaching with Active Dataset
+I(37007) OPENTHREAD:[N] Mle-----------: RLOC16 fffe -> 2801
+I(37007) OPENTHREAD:[N] Mle-----------: Role detached -> child
+
+```
+
+Set the role to router.
+
+```console
+> state router
+
+Done
+I(46057) OPENTHREAD:[N] Mle-----------: RLOC16 2801 -> 4400
+I(46057) OPENTHREAD:[N] Mle-----------: Role child -> router
+I(46057) OPENTHREAD:[N] Mle-----------: Partition ID 0x28b518c6
+> 
+```
+
+<img src="img/esp_ot_tp_LRR.jpg" alt="esp_ot_tp_LRR.jpg"  width="400"/>
+
+## Ping IPv6 address between Thread nodes
+
+Duration: 2:00
+
+Now you get a thread network with the topology shown blow:
+
+You can use ping command to communicate between any two boards. Use the command `ipaddr` to print each boards IPv6 address:
```

Comment:
"the ping command...."
boards should be board's - or rewrite as: "...to print the IPv6 address of each board."